docker/mongodb-kubernetes-init-database/content/agent-launcher.sh

```diff
@@ -145,11 +145,6 @@ else
 fi
 
 
-
-if [[ -n "${HTTP_PROXY-}" ]]; then
```

Comment:
nice! I wanted to remove this one ages ago. That just makes no sense - since the go lib gets those from the env var anyway